packages/fakta-medlemskap/src/v3/components/Divider.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
Kandidat for https://navikt.github.io/ft-frontend-saksbehandling/@navikt/ft-ui-komponenter/index.html ? Ser det ligg ein AvsnittSkiller der frå før. Er dette noko lignande denne?

Reply:
✅